pandas-stubs/core/reshape/encoding.pyi

```diff
@@ -3,20 +3,17 @@ from collections.abc import (
     Iterable,
 )
 
-from pandas import (
-    DataFrame,
-    Series,
-)
+from pandas import DataFrame
 
 from pandas._typing import (
-    ArrayLike,
+    AnyArrayLike,
     Dtype,
     HashableT1,
     HashableT2,
 )
 
 def get_dummies(
-    data: ArrayLike | DataFrame | Series,
+    data: AnyArrayLike | DataFrame,
```

Comment:
Index is supported here too, so I think `AnyArrayLike` could replace `ArrayLike` and `Series`?

Reply:
docs don't mention `Index`, so maybe open an issue there.  I'm fine with this change.